lib/xcode/install.rb

```diff
@@ -25,18 +25,19 @@ class Curl
     # @param progress: parse and show the progress?
     # @param progress_block: A block that's called whenever we have an updated progress %
     #                        the parameter is a single number that's literally percent (e.g. 1, 50, 80 or 100)
-    # @param retry_download_count: A count to retry the downloading Xcode dmg/xip
+    # @param number_of_try: How many times try to download DMG file if downloading fails. Default is 3.
     # rubocop:disable Metrics/AbcSize
     def fetch(url: nil,
               directory: nil,
               cookies: nil,
               output: nil,
               progress: nil,
               progress_block: nil,
-              retry_download_count: 3)
+              number_of_try: nil)
```

Comment:
Test fails by this default variable because `nil` is invalid at ↓, how about changing to actual number?https://github.com/xcpretty/xcode-install/pull/405/files#diff-bdac50e7bcf1817aa8cc74aaaeffcdedR82

Reply:
or adding nil checking like other variable in `fetch`.

Reply:
You are right, fixed it.